.github/workflows/ci.yaml

```diff
@@ -6,10 +6,16 @@ on:
 
 jobs:
   unit-tests:
+    strategy:
+      matrix:
+        go:
+          - '=1.18'
+          - '^1.23'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: '>=1.18.0'
-      - run: go test
+          go-version: ${{ matrix.go }}
+      - run: go test -v ./...
+      - run: cd v2 && go test -v ./...
```

Comment:
Go will only run the tests for the current module directory and not for the modules inside subdirectories, so this means that without explicitly running for `v2`, the tests would run only for `github.com/elliotchance/orderedmap`, and not for `github.com/elliotchance/orderedmap/v2`.

Reply:
Thanks for fixing this!